app/models/page_view.rb

```diff
@@ -38,6 +38,9 @@ class PageView < ApplicationRecord
 
     attributesForFaceting ["filterOnly(viewable_by)"]
 
+    attributeForDistinct :article_path
```

Comment:
Will this be distinct on `article_path` on a per-user basis? It seems like this would be a global distinct, which wouldn't be correct eh?

Reply:
Great catch, I think you're right, I thought it took into account the fact that the items are faceted per user (see previous line) but I guess it's a global attribute, so we need to find another attribute for the de-duplication

Reply:
I double checked, the de-duplication is done at query time (not at at the indexing time), so the articles are de-duplicated for the querying user (taking into account `viewable_by` which is set per each query). 

This is what happens after I browsed for a while with a second user:

![Screenshot 2019-06-25 at 1 36 26 PM](https://user-images.githubusercontent.com/146201/60095300-82403500-974e-11e9-8c79-9aa8140ee244.png)

If I disable distinct in JavaScript I guess all single visits:

![Screenshot 2019-06-25 at 1 37 28 PM](https://user-images.githubusercontent.com/146201/60095333-9a17b900-974e-11e9-8792-b0dcc0066530.png)

So yeah, since the de-duplication is done at query time and not during indexing, it works.

Reply:
> So yeah, since the de-duplication is done at query time and not during indexing, it works.

Ahh... That clarifies a lot for me.